### PR TITLE
Update 00-sql-introduction.md

### PR DESCRIPTION
**Pull request by chbrandt**
Rewrote the block about "relational database", it was somewhat vague and confusing.

### CODE REVIEW

**Comment by jd-foster** [comment]
Hi Carlos, thanks for taking the time to contribute to the introduction. 
I appreciate your efforts to help clarify the terminology around relational databases.
I am mindful of the lesson design principles we try to keep to, particularly [considering cognitive load](https://cdh.carpentries.org/developing-content.html#considering-cognitive-load). I think that there is a danger here of introducing concepts we might not need to use in the introduction. I wonder if your ideas could be included in later episodes where they are used?

That said, we could use some more motivation in the introduction, and your last two sentences:

> The idea of *relational data* becomes more and more important as the amount and complexity of data grows. And that is really the motivation for *relational databases*: to organize data in small, topic-related sets.
 
could fit in here with some tweaks.

File: episodes/00-sql-introduction.md [hunk 1]
```diff
@@ -104,12 +104,20 @@ Putting our data into a relational database and using SQL will help us achieve t
...
+on the scientific topics and what is being recorded in the database, the jargons change a little bit.
+
+The idea of *relational* is associated with organizing information across multiple tables in the database.
+The *relationship* between tables in the database can be established (or even enforced) in different ways,
+from a pure conceptual decision to the use of *keys* and other mechanisms.
+
+In many everyday datasets we don't need multiple tables and we don't have to think about *relations* 
+within data sets. 
+The idea of *relational data* becomes more and more important as the amount and complexity of data grows.
+And that is really the motivation for *relational databases*: to organize data in small, topic-related sets.
```
Comment (jd-foster) [inline episodes/00-sql-introduction.md:118]:
```suggestion
And that is really the motivation for relational databases: to organize data in through related sets.
```

File: episodes/00-sql-introduction.md [hunk 2]
```diff
@@ -104,12 +104,20 @@ Putting our data into a relational database and using SQL will help us achieve t
...
+
+Records in one tables may contain keys that refer to records in other tables,
```
Comment (jd-foster) [inline episodes/00-sql-introduction.md:120]:
```suggestion
Records in one table may contain keys that refer to records in other tables,
```

**Comment by chbrandt** [comment]
Hey @jd-foster , I see your point. And thanks for the fixes. Do you think we should stay with the original text (and *maybe* keep the two sentences you refer to), or just simplify what's in this branch? By all means, if the former is what you prefer, feel very free to just reject this PR.

**Comment by jd-foster** [comment]
Sorry for the delay in response. How do you feel about simplifying the text based on the comments?

File: episodes/00-sql-introduction.md [hunk 3]
```diff
@@ -104,12 +104,20 @@ Putting our data into a relational database and using SQL will help us achieve t
 
 ### Definition: *Relational Database*
 
-A relational database stores data in *relations* made up of *records* with *fields*.
-The relations are usually represented as *tables*;
-each record is usually shown as a row, and the fields as columns.
-In most cases, each record will have a unique identifier, called a *key*,
-which is stored as one of its fields.
-Records may also contain keys that refer to records in other tables,
+A relational database stores data in *tables* made up of *records* (rows) and *attributes* (columns).
```
Comment (jd-foster) [inline episodes/00-sql-introduction.md:107]: Omit this line.

File: episodes/00-sql-introduction.md [hunk 4]
```diff
@@ -104,12 +104,20 @@ Putting our data into a relational database and using SQL will help us achieve t
...
+*Attributes* may also be called *fields*, as *records* may also be called *observations*; Depending
```
Comment (jd-foster) [inline episodes/00-sql-introduction.md:108]: Please omit.

File: episodes/00-sql-introduction.md [hunk 5]
```diff
@@ -104,12 +104,20 @@ Putting our data into a relational database and using SQL will help us achieve t
...
+on the scientific topics and what is being recorded in the database, the jargons change a little bit.
```
Comment (jd-foster) [inline episodes/00-sql-introduction.md:109]: Please omit this.

File: episodes/00-sql-introduction.md [hunk 6]
```diff
@@ -104,12 +104,20 @@ Putting our data into a relational database and using SQL will help us achieve t
...
+
+The idea of *relational* is associated with organizing information across multiple tables in the database.
```
Comment (jd-foster) [inline episodes/00-sql-introduction.md:111]: Please keep the remaining lines from here on.

File: episodes/00-sql-introduction.md [hunk 7]
```diff
@@ -104,12 +104,20 @@ Putting our data into a relational database and using SQL will help us achieve t
...
-A relational database stores data in *relations* made up of *records* with *fields*.
-The relations are usually represented as *tables*;
-each record is usually shown as a row, and the fields as columns.
-In most cases, each record will have a unique identifier, called a *key*,
-which is stored as one of its fields.
-Records may also contain keys that refer to records in other tables,
+A relational database stores data in *tables* made up of *records* (rows) and *attributes* (columns).
+*Attributes* may also be called *fields*, as *records* may also be called *observations*; Depending
+on the scientific topics and what is being recorded in the database, the jargons change a little bit.
+
```
Comment (jd-foster) [inline episodes/00-sql-introduction.md:110]:
```suggestion
In most cases, each record will have a unique identifier, called a *key*, which is stored as one of its fields.
```

File: episodes/00-sql-introduction.md [hunk 8]
```diff
@@ -104,12 +104,20 @@ Putting our data into a relational database and using SQL will help us achieve t
...
+The idea of *relational* is associated with organizing information across multiple tables in the database.
+The *relationship* between tables in the database can be established (or even enforced) in different ways,
+from a pure conceptual decision to the use of *keys* and other mechanisms.
+
+In many small datasets we have only a few tables and we don't have to think about cross-references
+within data sets. 
+The idea of *relational data* becomes more and more important as the amount and complexity of data grows.
+And that is really the motivation for relational databases: to organize data through related sets.
```
Comment (jd-foster) [inline episodes/00-sql-introduction.md:118]:
```suggestion
This provides the motivation for using relational databases: to organize data through related keys.
```

File: episodes/00-sql-introduction.md [hunk 9]
```diff
@@ -104,12 +104,20 @@ Putting our data into a relational database and using SQL will help us achieve t
...
-In most cases, each record will have a unique identifier, called a *key*,
-which is stored as one of its fields.
-Records may also contain keys that refer to records in other tables,
+A relational database stores data in *tables* made up of *records* (rows) and *attributes* (columns).
+*Attributes* may also be called *fields*, as *records* may also be called *observations*; Depending
+on the scientific topics and what is being recorded in the database, the jargons change a little bit.
+
+The idea of *relational* is associated with organizing information across multiple tables in the database.
+The *relationship* between tables in the database can be established (or even enforced) in different ways,
+from a pure conceptual decision to the use of *keys* and other mechanisms.
```
Comment (jd-foster) [inline episodes/00-sql-introduction.md:113]:
```suggestion
through the use of *keys* and other constraints.
```

File: episodes/00-sql-introduction.md [hunk 10]
```diff
@@ -104,12 +104,20 @@ Putting our data into a relational database and using SQL will help us achieve t
...
-each record is usually shown as a row, and the fields as columns.
-In most cases, each record will have a unique identifier, called a *key*,
-which is stored as one of its fields.
-Records may also contain keys that refer to records in other tables,
+A relational database stores data in *tables* made up of *records* (rows) and *attributes* (columns).
+*Attributes* may also be called *fields*, as *records* may also be called *observations*; Depending
+on the scientific topics and what is being recorded in the database, the jargons change a little bit.
+
+The idea of *relational* is associated with organizing information across multiple tables in the database.
+The *relationship* between tables in the database can be established (or even enforced) in different ways,
```
Comment (jd-foster) [inline episodes/00-sql-introduction.md:112]:
```suggestion
The *relationships* between tables in a database can be established (or even enforced) in different ways,
```

File: episodes/00-sql-introduction.md [hunk 11]
```diff
@@ -104,12 +104,20 @@ Putting our data into a relational database and using SQL will help us achieve t
...
+from a pure conceptual decision to the use of *keys* and other mechanisms.
+
+In many small datasets we have only a few tables and we don't have to think about cross-references
+within data sets.
```
Comment (jd-foster) [inline episodes/00-sql-introduction.md:116]:
```suggestion
within datasets. 
```

**Comment by jd-foster** [comment]
Hi @chbrandt. Anything I can help with here? I would like to improve the text using your PR; are you happy if I open a new PR based on this one?

**Comment by chbrandt** [comment]
Hey @jd-foster. 
I figured the specifics of my suggestions were not really adding much to the discussion, so I closed it. 
But, sure, you are very welcome to use the PR however you want.

**Pull request closed without merging**